scss/_dropdown.scss

```diff
@@ -9,14 +9,14 @@
 }
 
 .dropdown-toggle {
+  // Boosted mod
   // scss-docs-start dropdown-toggle-css-vars
-  --#{$prefix}dropdown-toggle-padding-x: #{$dropdown-padding-x}; // Boosted mod
```

Comment:
Don't you think it is important that the `// Boosted mod` comments appear in the docs?
And then that some should be added as well lines 29 to 33?

Reply:
Ah yes you're right, I totally forgot about the doc!